src/typedefs.rs

```diff
@@ -750,7 +750,7 @@ macro_rules! construct_secret_key_variable_size {
         /// # }
         /// ```
         pub struct $name {
-            value: Vec<u8>,
+            pub(crate) value: Vec<u8>,
```

Comment:
This is set to `pub(crate)` so we avoid allocation two different `Vec<u8>`s in `orion::kdf` which hold sensitive data.